fastlane/lib/fastlane/helper/git_helper.rb

```diff
@@ -3,36 +3,38 @@ module Actions
     GIT_MERGE_COMMIT_FILTERING_OPTIONS = [:include_merges, :exclude_merges, :only_include_merges].freeze
 
     def self.git_log_between(pretty_format, from, to, merge_commit_filtering, date_format = nil, ancestry_path)
-      command = ['git log']
-      command << "--pretty=\"#{pretty_format}\""
-      command << "--date=\"#{date_format}\"" if date_format
+      command = %w(git log)
+      command << "--pretty=#{pretty_format}"
+      command << "--date=#{date_format}" if date_format
       command << '--ancestry-path' if ancestry_path
-      command << "#{from.shellescape}...#{to.shellescape}"
+      command << "#{from}...#{to}"
       command << git_log_merge_commit_filtering_option(merge_commit_filtering)
-      Actions.sh(command.compact.join(' '), log: false).chomp
+      Actions.sh(*command.compact, log: false).chomp
```

Comment:
What does the `*` indicate here? (Yep, my ruby is rudimentary)

Reply:
Ok, so before we used a joined string as a param, not the "vararged" array.

Reply:
Is this common enough that most ruby devs will know about this? Otherwise maybe a simple comment above that line?

Reply:
Exactly.

It's been a while since I've been paid for writing Ruby code, so I can't tell how common it is nowadays. Note that it's also used in `Actions.sh` itself (both argument list and a method call). I added a comment nevertheless, better safe than sorry.

https://github.com/fastlane/fastlane/blob/30be4096f3c9a84c9fee3cf4a73c71a7508313f4/fastlane/lib/fastlane/helper/sh_helper.rb#L11-L13

Reply:
Thanks.

`sh_helper.rb` has been written by @jdee whose ruby is pretty advanced sometimes (at least compare to what I produce) ;) But I think we are good here as well now.